views/courses/create-course.hbs

```diff
@@ -16,9 +16,33 @@
         <script src="/scripts/spectrum/spectrum.js" type="text/javascript" nonce="{{nonceValue}}" defer></script>
         <script src="/scripts/spectrum/init.js" type="text/javascript" nonce="{{nonceValue}}" defer></script>
         <script src="/scripts/dataprivacy/dataprivacy.js" type="text/javascript" nonce="{{nonceValue}}" defer></script>
+        <script nonce="{{nonceValue}}">
+            // a workaround to validate a chosen-generated dropdown (multiselect) which is not validated out of the box
+            window.addEventListener('DOMContentLoaded', function () {
```

Comment:
we need to have a mechanism to mark chosen as required and to show an error if there is no value.
It should have a general solution which will work for all select input fields.